core/integration/container_test.go

```diff
@@ -2805,6 +2805,34 @@ func (ContainerSuite) TestPublish(ctx context.Context, t *testctx.T) {
 	require.Equal(t, "im-a-default-arg\n", output)
 }
 
+func (ContainerSuite) TestAnnotations(ctx context.Context, t *testctx.T) {
```

Comment:
Test is failing because, for some reason, the annotations only end up in multi platform images, not single platform

/cc @jedevc